README.md

```diff
@@ -24,28 +24,27 @@ Any changes made by this module affect the current `running-config`. These chang
 
 ## Setup
 
-### Setup Requirements
+**Device Connection:** This module requires a user that can access the device via SSH and that has the `enable mode` privilege.
 
-This module requires a user that can access the device via SSH and that has the `enable mode` privilege.
+**Proxy Agent Node:** Since there is no agent available for the Catalysts (and, seriously, who would want to run an agent on them?) we need to run puppet on a proxy agent node (either a compile master, or regular puppet agent install).
 
-### Beginning with cisco_ios
-
-To get started, create or edit `/etc/puppetlabs/puppet/device.conf`, add a section for the device (this will become the device's `certname`), specify a type of `cisco_ios`, and specify a `url` to a credentials file. For example:
-
-```INI
-[cisco.example.com]
-type cisco_ios
-url file:////etc/puppetlabs/puppet/devices/cisco.example.com.conf`
+**Install dependencies:** Declare the `cisco_ios` class on the proxy agent node to deploy the required dependencies to run this content:
```

Comment:
```suggestion
**Install dependencies:** Declare the `cisco_ios` class on the proxy Puppet agent to install any dependencies required by this module:
```

Reply:
Actually, this is not necessary when using the `device_manager` module, as it includes the class by default ...

https://forge.puppet.com/puppetlabs/device_manager#include_module